### DISCUSSION TITLE
libxsmm doesn't support intel compiler?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#762] [Q&A] [answered]
when I try to compile with intel compiler ,I encouter this problem
![image](https://user-images.githubusercontent.com/114474894/208230408-50403281-dc72-4183-99c4-71969188077e.png)
when I use export CC=gcc  CXX=g++  FC=gfortran ,it works nicely.
![image](https://user-images.githubusercontent.com/114474894/208230446-b15cd9b8-2bad-4da7-8c8f-1e35d5df7423.png)

**Comment by sebwolf-de** [comment, marked as answer]
Intel Compiler 13 is too old to compile SeisSol:
`Intel compiler (>= 2021, icc, icpc, ifort) or GCC (>= 9.0, gcc, g++, gfortran)` (https://seissol.readthedocs.io/en/latest/installing-dependencies.html)

**Reply by buzhiqiuyuebai** [reply]
Thanks a lot, I compiled SeisSol use gcc before, so I didn't notice the icc version issue.
